config/config.go

```diff
@@ -6,11 +6,12 @@ import (
 
 // Config -
 type Config struct {
-	Project  string
-	Zones    []string
-	Regions  []string
-	Timeout  int
-	PollTime int
-	Context  context.Context
-	DryRun   bool
+	Ctx                      context.Context
```

Comment:
I think `Context` would actually be better here, fwiw, as it's a struct field not a function argument (e.g. even in the context package itself it's named `Context`, not `Ctx`)

Reply:
Surely the convention is to name variables holding `context.Context` to is `ctx`? Being a public struct member then it becomes `Ctx`?
I'm happy to revert, but official Go examples always use `ctx`

Reply:
From the documentation:
Do not store Contexts inside a struct type; instead, pass a Context explicitly to each function that needs it. The Context should be the first parameter, typically named ctx:
```
func DoSomething(ctx context.Context, arg Arg) error {
	// ... use ctx ...
}
```

Reply:
Of course it says not to store in a `struct` type, so I guess the rules can be bend somewhat! :-)

Reply:
Yes, storing context in a struct is bad practice actually, it would be better if we didn't do that (but I assume that's a bigger change than just renaming)